pkgs/tools/misc/turbo/default.nix

```diff
@@ -64,7 +65,8 @@ let
     pname = "go-turbo";
     modRoot = "cli";
 
-    vendorHash = "sha256-8quDuT8VwT3B56jykkbX8ov+DNFZwxPf31+NLdfX1p0=";
+    proxyVendor = true;
```

Comment:
Without it fails with:
```
go: -mod may only be set to readonly when in workspace mode, but it is set to "vendor"
```

Reply:
I think the better option is to (get upstream to) delete the `go.work` file they added in vercel/turbo#6111. Being a single-module workspace, this has no benefits and is arguably wrong.

Until then, it can be easily circumvented in nixpkgs with something like [this](https://github.com/NixOS/nixpkgs/compare/master...nevivurn:nixpkgs:fix/turbo-1.11.2?expand=1#diff-1749a25f8e5b5e45350b1a62a995646926e8b6addc00d169edd5e80888f56c3fR84).